### PR TITLE
[docs]: Readme

### LINKED ISSUE #2: Update Readme
A more professional readme would be nice. 

Mind if I open a PR for that?

### PR DESCRIPTION
**Pull request by appetrosyan**
Closes #2.

### CODE REVIEW

**Comment by appetrosyan** [comment]
@ragnard Given that I'm neither a collaborator, you don't have `CODEOWNERS` I'd like to formally request your review.

File: README.md [hunk 1]
```diff
@@ -2,8 +2,74 @@
...
+(setq tabby-api-url "http://localhost:8080")
+```
+</details>
+
 ## Usage
 
-- Install from [MELPA](https://melpa.org)
-- Set `tabby-api-url` to the URL of your Tabby server
-- Use `tabby-complete` to fetch completion suggestions from Tabby
+Manually invoke `tabby-complete`.
```
Comment (ragnard) [inline README.md:65]: It is actually bound to `C-<tab>` in the default keymap now, so maybe mention that?

Comment (ragnard) [inline README.md]: It is actually bound to `C-<tab>` in the default keymap now, so maybe mention that?

**Review by ragnard — COMMENTED** [review]
Thanks for this, looks good but see comment about key-binding.

File: README.md (hunk 1 above)
Comment (appetrosyan) [reply README.md]: Yep, easy

Comment (appetrosyan) [reply README.md]: Done

**Comment by appetrosyan** [comment]
Merge?

**Pull request closed without merging**